examples/flutter_gallery/lib/demo/animation_demo.dart

```diff
@@ -272,12 +276,21 @@ class _RectangleDemoState extends State<_RectangleDemo> {
 
   CurvedAnimation _animation;
   _DragTarget _dragTarget;
-  Rect _begin = new Rect.fromLTRB(180.0, 100.0, 330.0, 200.0);
-  Rect _end = new Rect.fromLTRB(32.0, 275.0, 132.0, 425.0);
+  Rect _begin;
+  Rect _end;
 
   @override
   void initState() {
     super.initState();
+    final Size screenSize = MediaQuery.of(context).size;
```

Comment:
Does that mean your LGTM still stands?  Is there a better way to do this example?